lib/utils/host/hostusers.go

```diff
@@ -65,7 +67,7 @@ func UserAdd(username string, groups []string, home, uid, gid string) (exitCode
 	}
 
 	if home == "" {
-		return -1, trace.BadParameter("home is a required parameter")
+		home = string(os.PathSeparator)
 	}
```

Comment:
Definitely a nitpick, but maybe worth a comment here explaining why we default users to `/`? Just seemed a bit surprising, though I can't see any problem with it.